static/_redirects

```diff
@@ -5,20 +5,20 @@
 ###############################################
 
 /concepts/containers/container-lifecycle-hooks/     /docs/concepts/containers/container-lifecycle-hooks/ 301
-/docs/     /docs/home/ 301!
-/de/docs/     /de/docs/home/ 301!
-/es/docs/     /es/docs/home/ 301!
-/fr/docs/     /fr/docs/home/ 301!
-/id/docs/     /id/docs/home/ 301!
-/ja/docs/     /ja/docs/home/ 301!
-/ko/docs/     /ko/docs/home/ 301!
-/no/docs/     /no/docs/home/ 301!
-/pl/docs/     /pl/docs/home/ 301!
-/pt/docs/     /pt/docs/home/ 301!
-/ru/docs/     /ru/docs/home/ 301!
-/uk/docs/     /uk/docs/home/ 301!
-/vi/docs/     /vi/docs/home/ 301!
-/zh/docs/     /zh/docs/home/ 301!
+#/docs/     /docs/home/ 301!
+#/de/docs/     /de/docs/home/ 301!
+#/es/docs/     /es/docs/home/ 301!
+#/fr/docs/     /fr/docs/home/ 301!
+#/id/docs/     /id/docs/home/ 301!
+#/ja/docs/     /ja/docs/home/ 301!
+#/ko/docs/     /ko/docs/home/ 301!
+#/no/docs/     /no/docs/home/ 301!
+#/pl/docs/     /pl/docs/home/ 301!
+#/pt/docs/     /pt/docs/home/ 301!
+#/ru/docs/     /ru/docs/home/ 301!
+#/uk/docs/     /uk/docs/home/ 301!
+#/vi/docs/     /vi/docs/home/ 301!
+#/zh/docs/     /zh/docs/home/ 301!
```

Comment:
We still need to keep these redirects.

Reply:
But if we keep these rules, we won't let user see the `/docs/` page, that's the reason that I disable them :(

Reply:
If we enable these rules, we could **ONLY** print out the sub-sections under the docs, since users couldn't visit the `/docs/` page ( it would redirect to `/docs/home/` automatically ).

Reply:
@sftim It's also okay to me if we choose to keep these rules and just let users could print out each sub-section pages, e.g. [Concepts](https://deploy-preview-26809--kubernetes-io-master-staging.netlify.app/docs/concepts/_print/), [Tasks](https://deploy-preview-26809--kubernetes-io-master-staging.netlify.app/docs/tasks/_print/).

After all, we could now read the k8s docs via iPad or Web Browser by PDF(via web print) or Chrome, it's great to some people, including me :)

Reply:
> let users could print out each sub-section pages, e.g. [Concepts](https://deploy-preview-26809--kubernetes-io-master-staging.netlify.app/docs/concepts/_print/), [Tasks](https://deploy-preview-26809--kubernetes-io-master-staging.netlify.app/docs/tasks/_print/).

Sounds good to me @Colstuwjx